src/WebJobs.Script.WebHost/Diagnostics/Extensions/ScriptHostServiceLoggerExtension.cs

```diff
@@ -323,5 +329,10 @@ public static void ScriptHostServiceRestartCanceledByRuntime(this ILogger logger
         {
             _scriptHostServiceRestartCanceledByRuntime(logger, null);
         }
+
+        public static void LogSettings(this ILogger logger, string functionWorkerRuntime, string functionExtensionVersion, string currentDirectory, string inStandbyMode)
```

Comment:
also rename to `LogHostInitializationSettings` (assuming you like that name) -- and that'll update the event name as well.

Reply:
changed!